### PR TITLE
[PAIR] Profile Picture Datastore & Insertion

### PR DESCRIPTION
**Pull request by lalaAdasia00**
Screenshot 2020-07-24 at 11 48 45 AM - Display 2

### CODE REVIEW

**Comment by lalaAdasia00** [comment]
The "some change" commit was just to help someone with basic git commands

File: html/user.hbs [hunk 1]
```diff
@@ -1,6 +1,9 @@
-<script src="/scripts/share.js" type="module"></script>
+<script src="/scripts/auth.js" type="module"></script>
```
Comment (ifeomi) [inline html/user.hbs:1]: does share.js need to be included somewhere else?

Comment (lalaAdasia00) [reply html/user.hbs:1]: no that was accidentally deleted. It needs to be on the user.hbs

**Review by ifeomi — APPROVED** [review]

**Review by ifeomi — COMMENTED** [review]
there is a conflict with auth.js

**Comment by lalaAdasia00** [comment]
> there is a conflict with auth.js

Just fixed :)

**Review by chronologos — APPROVED** [review]

**Pull request merged**